app/src/main/cpp/src/components/transport_manager/include/transport_manager/bluetooth_le/ble_control_server.h

```diff
@@ -0,0 +1,31 @@
+#ifndef SRC_COMPONENTS_TRANSPORT_MANAGER_INCLUDE_TRANSPORT_MANAGER_BLUETOOTH_LE_BLE_CONTROL_SERVER_H_
+#define SRC_COMPONENTS_TRANSPORT_MANAGER_INCLUDE_TRANSPORT_MANAGER_BLUETOOTH_LE_BLE_CONTROL_SERVER_H_
+#include <stdio.h>
+#include <stdlib.h>
+#include <unistd.h>
+#include <string.h>
+#include <sys/types.h>
+#include <sys/socket.h>
+#include <sys/un.h>
+#include <netinet/in.h>
+#include <netdb.h>
+#include <mutex>
```

Comment:
I believe we don't need mutex here

Reply:
done in 66d56ca414ac2d17dea823a2186a8d3f2569ec83